app/models/manageiq/providers/amazon/cloud_manager.rb

```diff
@@ -202,13 +202,14 @@ def self.discover(access_key_id, secret_access_key)
     known_emses = all_emses.select { |e| e.authentication_userid == access_key_id }
     known_ems_regions = known_emses.index_by(&:provider_region)
 
-    ec2 = raw_connect(access_key_id, secret_access_key, "EC2")
-    ec2.regions.each do |region|
-      next if known_ems_regions.include?(region.name)
-      next if region.instances.count == 0 && # instances
-              region.images.with_owner(:self).count == 0 && # private images
-              region.images.executable_by(:self).count == 0  # shared  images
-      new_emses << create_discovered_region(region.name, access_key_id, secret_access_key, all_ems_names)
+    ec2 = raw_connect_v2(access_key_id, secret_access_key, "EC2", "us-east-1")
```

Comment:
@Fryguy I am about to rework this to use `with_provider_connection` to remove this raw_connect here. And then `raw_connect_v2` will default to the provider_region set on the ems/cloudmanager.

In v2 of the SDK you have to provide a region for making connections. This discovery code works by enumerating over all regions returned by the api, but we have to provide one.